src/patternfly/components/Content/content.scss

```diff
@@ -148,69 +146,87 @@
   }
 
   h1 {
-    margin-bottom: var(--pf-c-content--h1--MarginBottom);
     font-size: var(--pf-c-content--h1--FontSize);
     font-weight: var(--pf-c-content--h1--FontWeight);
     line-height: var(--pf-c-content--h1--LineHeight);
 
     &:not(:first-child) {
       margin-top: var(--pf-c-content--h1--MarginTop);
     }
+
+    &:not(:last-child) {
```

Comment:
What if you set
```
  h1 {
    margin-top: var(--pf-c-content--h1--MarginTop);
    margin-bottom: var(--pf-c-content--h1--MarginBottom);
  }

  h2 {
    margin-top: var(--pf-c-content--h2--MarginTop);
    margin-bottom: var(--pf-c-content--h2--MarginBottom);
  }

  h1,
  h2,
  h3,
  h4 {
    &:first-child {
      margin-top: 0;
    }

    &:last-child {
      margin-bottom: 0;
    }
  }
```

Reply:
That would work, nice idea

Reply:
Is there any downside to setting it and then removing it, no right

Reply:
Not that I can see:D